data/mods/gen9ssb/moves.ts

```diff
@@ -1611,12 +1611,25 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 				msg = "We asked an AI to make a randbats set. YOU WON'T BELIEVE WHAT IT CAME UP WITH N.N";
 				break;
 			}
-			// TODO: ban mons with custom stats
-			const team = Teams.generate(randFormat, {name: target.side.name});
-			this.addMove('-anim', target, 'Wish', target);
-			// @ts-ignore set wants a sig but randbats sets don't have one
-			changeSet(this, target, team[0], true);
-			this.add(`c:|${getName((source.illusion || source).name)}|${msg}`);
+			while (true) {
```

Comment:
infinite loop!

Reply:
it's _technically_ not infinite since there's a break at the bottom reached after a successful pokemon is found.

Reply:
anyway if you'd prefer to take this, go for it